### DISCUSSION TITLE
Simple Helper GUI to start/stop the current time from Windows

### DISCUSSION THREAD

**Discussion by leonbohmann** [kimai/kimai#4886] [Plugins]
Hey all, just a quick hint to a really small helper software.

I was searching the internet for a small program I can use to track the working time. For me, it was too complicated to always go to the web-ui to create the item, then come back another time and edit the description, insert the working hours and so on.

Therefore I created a prototype: [timer](https://github.com/leonbohmann/timer)

Let me know what you think! Is that something you would use as well?
In the future I am thinking about further integrating the API-Features and restructuring the code, it is a little clumsy at the moment 😜

**Comment by kevinpapst** [comment]
Interesting, can you share a short video here?
I am not a windows user, so I can't test.

If you want, you can use https://github.com/kimai/images

**Reply by leonbohmann** [reply]
Sure, here is a description of the steps :)
Also, I created a publish profile for linux, just in case you are working on linux?


## When starting, insert the server adress and your token:
![aiw9K8u6ow](https://github.com/kimai/kimai/assets/13386367/fd39137a-b88f-456a-8f5e-ced0bdd81fab)

## Then, write something and start the action
![lADcG1QbGN](https://github.com/kimai/kimai/assets/13386367/035d0a46-af96-414b-a8db-7420fe924faa)

## In Kimai, the changes are reflected
![grafik](https://github.com/kimai/kimai/assets/13386367/c813528b-47cf-47af-a07a-b8ec47ad43c9)

## Stop the entry
![JYyNOK66qs](https://github.com/kimai/kimai/assets/13386367/5059140a-b63a-45a6-b9a9-203928d233c2)

## See reflected changes and correct duration on the item
![grafik](https://github.com/kimai/kimai/assets/13386367/8a380d48-4e96-4bea-b974-d14513e30d26)

**Reply by leonbohmann** [reply]
Well, so far this is only designed to look good with 1 or 2 projects and activities. If there are more entries to display, the layout will probably look not so good.

Maybe, if that is of interest to anyone, I could oneday fix the layout! For my needs this is enough, because I only want to track the amount of time at work and in homeoffice.

**Reply by kevinpapst** [reply]
Mac user, seems dotnet is available there as well, I just never had a reason to install it before.

Lets see if people are interested!

**Reply by leonbohmann** [reply]
Btw, I really like the implementation of Kimai. Not seeing a well implemented solution that often! Good Work!
